pool/position_modify.gno

```diff
@@ -11,13 +11,7 @@ import (
 // modifyPosition updates a position in the pool and calculates the amount of tokens to be added or removed.
 // Returns positionInfo, amount0, amount1
 func (pool *Pool) modifyPosition(params ModifyPositionParams) (PositionInfo, *i256.Int, *i256.Int) {
```

Comment:
This func should be simply about updating position. You can divide calculateTokenDeltas into another function.

Reply:
Checked you already did this in #402. Great job.